coalib/settings/Setting.py

```diff
@@ -108,7 +125,7 @@ def __init__(self,
         self.key = key
         self.origin = str(origin)
 
-    def __path__(self, origin=None):
+    def __path__(self, origin=None, globbed_origin=False):
```

Comment:
and if this param is for setting whether to interprete `origin` as glob or not, I would rename the parameter, this name is not so obvious. Maybe a plain `is_glob`?